angularSTEP/src/app/upload-recipe/upload-recipe.component.html

```diff
@@ -31,6 +31,35 @@ <h5 class="other-error" *ngIf="basicsFormGroup.controls.name.value.trim() === ''
           <mat-hint *ngIf="!(basicsFormGroup.controls.description.value.trim() === '' && basicsFormGroup.controls.description.touched && basicsFormGroup.controls.description.valid)">{{description.value?.length}}/200 Characters</mat-hint>
           <mat-error>This is required.</mat-error>
         </mat-form-field>
+
+        <br><br>
+        
+        <mat-form-field class="example-chip-list">
+          <mat-chip-list #chipList aria-label="Category selection">
+            <mat-chip
+              *ngFor="let category of categories"
+              [selectable]="selectable"
+              [removable]="removable"
+              (removed)="remove(category)">
+              {{category}}
+              <mat-icon matChipRemove *ngIf="removable">cancel</mat-icon>
+            </mat-chip>
+            <input
+              placeholder="New category..."
+              #fruitInput
+              [formControl]="categoryCtrl"
+              [matAutocomplete]="auto"
+              [matChipInputFor]="chipList"
+              [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+              (matChipInputTokenEnd)="add($event)">
+          </mat-chip-list>
+          <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selected($event)">
+            <mat-option *ngFor="let category of filteredCategories | async" [value]="catagory">
+              {{category}}
+            </mat-option>
+          </mat-autocomplete>
+        </mat-form-field>
+
         <h5 class="other-error" *ngIf="basicsFormGroup.controls.description.value.trim() === '' && basicsFormGroup.controls.description.touched && basicsFormGroup.controls.description.valid">Please enter a valid description.</h5>
```

Comment:
This line (the `<h5>` element with the class "other-error") is supposed to go directly after the closing tag `</mat-form-field>` or in other words should be line # 34

Reply:
Hey I think I resolved it correctly if you want to take a look at it before I merge just to make sure that would be great.